atlasdb-cassandra/src/test/resources/testConfigCQL.yml

```diff
@@ -0,0 +1,11 @@
+type: "cassandra"
+servers:
+  type: "cqlEnabled"
+  thrift:
+    - foo:42
```

Comment:
I agree with the format you propose here, it is definitely a better solution.

Reply:
I implemented the format @felixdesouza proposed, I will in any case check what's discoverable as per @jeremyk-91 proposed.

Reply:
Because of internal configuration reasons the format is changed from:

```yaml
servers:
  type: "cqlEnabled"
  hosts:
    - hostname: cassandra1
      thrift-port: 9160
      cql-port: 9042
    - hostname: cassandra2
      thrift-port: 9160
      cql-port: 9042   
    - hostname: cassandra3
      thrift-port: 9160
      cql-port: 9042   
```
to

```yaml
servers:
  type: "cqlEnabled"
  hosts:
    - cassandra1
    - cassandra2
    - cassandra3
  cqlPort: 9042
  thriftPort: 9160
```